test/integration/short/pool-simulator-tests.js

```diff
@@ -87,6 +88,89 @@ describe('pool', function () {
         .then(() => validateStateOfPool(client));
     });
   });
+
+  context('with a simulated cluster containing three nodes on each of the two datacenters', () => {
+    let cluster;
+
+    beforeEach(done => {
+      cluster = new simulacron.SimulacronCluster();
+      cluster.register([3, 3], null, done);
+    });
+
+    afterEach(done => cluster.unregister(done));
+
+    it('should not attempt to make a connection on a new node that is on an ignored/remote DC', () => {
+      // For the purpose of this test, we will use an address translator that will create the impression
+      // that on the second (ignored) DC, there are 2 nodes instead of 3
+      // Then, the address translator will show 3 nodes, giving the impression that the third node is "new".
```

Comment:
I might be missing something, but imho it would be easier to use the `DELETE` method of  `/connections/{clusterId}/{dataCenterId}/{nodeId}` of simulacron.